src/discord.ts

```diff
@@ -36,4 +40,40 @@ export default class Discord extends AinftBase {
     const trailingUrl = `${discordServerId}/${discordChannelId}/tasks`;
     return this.sendRequest(HttpMethod.GET, trailingUrl, query);
   }
+
+  addPersonaModelToDiscordChannel(
+    appId: string,
+    modelId: string,
+    discordServerId: string,
+    discordChannelId: string
+  ): Promise<void> {
+    const body = {
+      appId,
+      modelId,
+      discordServerId,
+      discordChannelId,
+    };
+    const trailingUrl = `/persona-models`;
```

Comment:
`/` is unnecessary. ref: https://github.com/ainft-team/ainft-js/blob/main/src/ainftBase.ts#L51

Reply:
I will fix it. Thank you!